error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1232,6 +1232,24 @@ StepVerifier.Step<T> after(StepVerifier.Step<T> step, T object) {
     }
   }
 
+  /**
+   * Prefer {@link StepVerifier.LastStep#verifyComplete()} over {@code
+   * StepVerifier.Step#expectNextCount(0).verifyComplete()} or its alternatives.
+   */
+  static final class StepVerifierStepExpectNextCountZeroVerifyComplete<T> {
+    @BeforeTemplate
+    Duration before(StepVerifier.Step<T> step) {
+      return Refaster.anyOf(
+          step.expectNextCount(0L).expectComplete().verify(),
+          step.expectNextCount(0L).verifyComplete());
```

Comment:
I think we're not yet generic enough: Ideally, we rewrite:
```java
          step.expectNextCount(0L)
```
to just 
```java
          step
```

After all, it's not guaranteed that the next call will be `verifyComplete()`. :eyes:

Reply:
Perfect ❤️ 
Thanks for heads-up 🕺